data/korath/korath missions.txt

```diff
@@ -257,3 +257,81 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Efret: Quarg Rescue"
+	boarding
+	repeat
+	invisible
+	deadline 1
+	source
+		government "Kor Efret"
+	on offer
+		"old quarg rep" = "reputation: Quarg"
+		# bring it low enough that capping this will result in war
+		"reputation: Quarg" <?= 1
```

Comment:
The Quarg's attitude towards the Efreti is 0.01. This means they won't actually care if I accept their warning and then capture the ship anyway.

Reply:
Oh well thats easy to fix but I didnt think it would be 0.01